models/hl7-v3/dispense-notification.ts

```diff
@@ -38,7 +36,7 @@ export class DispenseNotification implements ElementCompact {
       "163541000000107",
       "Dispensed Medication - FocusActOrEvent (administrative concept)"
     )
-    this.effectiveTime = convertMomentToHl7V3DateTime(moment.utc())
+    this.effectiveTime = new core.Timestamp("PLACEHOLDER")
```

Comment:
What's the reason for putting this back?

Reply:
because this is now in models and the `convertMomentToHl7V3DateTime` function is in `coordinator`, so it's constructed as a placeholder to avoid circular dependency and then actually set in the translation in coordinator